### ISSUE TITLE
blackout period API reference wrong

### ISSUE THREAD

**Issue by satterly** [alerta/alerta-docs#58]
create and list blackout endpoints should show service and tags as lists

**Comment by satterly** [comment]
```
{
  "blackout": {
    "customer": "Tyrell Corp. \u2021", 
    "duration": 3600, 
    "endTime": "2017-09-06T22:00:00.000Z", 
    "environment": "Host", 
    "id": "1f343ff0-2ffe-48d5-b26f-b0e096e4ad7c", 
    "priority": 3, 
    "service": [
      "Linux servers"
    ], 
    "startTime": "2017-09-06T21:00:00.000Z"
  }, 
  "id": "1f343ff0-2ffe-48d5-b26f-b0e096e4ad7c", 
  "status": "ok"
}
```

**Comment by satterly** [comment]
```
{
  "blackout": {
    "duration": 3600, 
    "endTime": "2017-09-06T22:00:00.000Z", 
    "environment": "Development", 
    "id": "dada07fb-8ea9-4e5a-b2cf-88d6c2f94338", 
    "priority": 7, 
    "startTime": "2017-09-06T21:00:00.000Z", 
    "tags": [
      "london", 
      "dc1"
    ]
  }, 
  "id": "dada07fb-8ea9-4e5a-b2cf-88d6c2f94338", 
  "status": "ok"
}
```

**Comment by satterly** [comment]
Fixed by #70
